src/GlobalStyle.js

```diff
@@ -0,0 +1,23 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyle = createGlobalStyle`
+  * {
+    box-sizing: border-box;
+  }
+  body {
+    margin: 0;
+    background-color: #12121a;
+    color: #ccd;
+    font-family: "Poppins", -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+      'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+      sans-serif;
+    -webkit-font-smoothing: antialiased;
```

Comment:
Do your really need the webkit and the moz properties?

Reply:
No i don't really need that... thx 👍🏻